multi_project/core/src/main/java/multi_project/core/Utils.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package multi_project.core;
+
+import utils.StringUtils;
+
+public class Utils {
+    public static boolean isAllPositiveNumbers(String... str) {
+        boolean isPositive = true;
+        for (String s : str) {
+            if (!StringUtils.isPositiveNumber(s)) {
+                isPositive = false;
+            }
+        }
+        return isPositive;
```

Comment:
Rewrite this using Stream API. Use `allMatch` method

Reply:
done